src/config/updateExtremes.js

```diff
@@ -19,7 +19,7 @@ export const updateExtremesXAxis = (chart, contract = {}, rangeButton) => {
 
     const dataFromChart = series.options.data;
     const lastTickMillis = dataFromChart[dataFromChart.length - 1] && dataFromChart[dataFromChart.length - 1][0];
-    const startTimeEpoch = (contract && contract.is_forward_starting) && contract.date_start;
```

Comment:
Was that custom added by us? Or it was the thing that backend changed? Or something else?